src/components/Expense.js

```diff
@@ -17,11 +13,11 @@ const Expense = ({ expense, showAlert }) => {
         <div className="col-8 col-lg-10">
           <div className="row align-items-center">
             <div className="col-lg-2 fs-5 fw-bold text-nowrap">
-              {getUKFormattedDate(date)}
+              {expense.getFormattedDate('en-GB')}
```

Comment:
this reads much nicer now. maybe the naming could be even more concise (e.g. expense.formattedDate('en-GB')